components/nds-card/src/Card.js

```diff
@@ -23,42 +23,64 @@ const CardHeader = props => {
 	}
 };
 
+const CardBody = props => {
+	const { metadata, headingText, headingElementType, link, summary } = props;
+	const headerProps = { headingText, headingElementType, link };
+	return (
+		<>
+			<header className="card__header">
+				<CardHeader {...headerProps} />
+			</header>
+			<p className="card__summary">{summary}</p>
```

Comment:
Shouldn't this be this so it only renders it if it's there?
```suggestion
			{summary && <p className="card__summary">{summary}</p>}
```